src/Symfony/Component/PasswordHasher/Tests/Command/UserPasswordHashCommandTest.php

```diff
@@ -67,7 +67,7 @@ public function testEncodePasswordBcrypt()
 
     public function testEncodePasswordArgon2i()
     {
-        if (!($sodium = SodiumPasswordHasher::isSupported() && !\defined('SODIUM_CRYPTO_PWHASH_ALG_ARGON2ID13')) && !\defined('PASSWORD_ARGON2I')) {
+        if (!($sodium = SodiumPasswordHasher::isSupported() && !\defined('SODIUM_CRYPTO_PWHASH_ALG_ARGON2ID13'))) {
```

Comment:
we can remove the outer brackets, isn't it? (same below)

Reply:
Not with the current code, plz look at my suggestion.